app/src/bat_charger.h

```diff
@@ -457,14 +645,21 @@ class Charger
      *
      * Must be called exactly once per second, otherwise SOC calculation gets wrong.
      */
-    void update_soc(BatConf *bat_conf);
+    void update_soc(BatConf *bat_conf, EkfSoc *ekf_soc);
+
+    /**
+     * SOC estimation
+     *
+     * WARNING: TODO obsolte function, replaced by void charge_control(BatConf *bat_conf);
```

Comment:
Isn't it replace by the new `update_soc` function with the additional EKF parameter?

Reply:
correct.